addPlant.php

```diff
@@ -0,0 +1,137 @@
+<?php
+            require_once 'src/PlantModel.php';
+
+            if (isset($_POST['newPlantName'])&&
+                isset($_POST['newPlantScientificName'])&&
+                isset($_POST['newPlantFamily'])&&
+                isset($_POST['newPlantImage'])&&
+                isset($_POST['newPlantDescription'])){
+
+                $inputtedPlantName = $_POST['newPlantName']; 
+                $inputtedScientificName = $_POST['newPlantScientificName'];
+                $inputtedPlantFamily = $_POST['newPlantFamily'];
+                $inputtedImage = $_POST['newPlantImage'];
+                $inputtedDescription = $_POST['newPlantDescription'];
+                
+                if(strlen($inputtedPlantName)==0 ||
+                is_numeric($inputtedPlantName)) {
+                    echo "Please select use valid characters for the plant name.";
```

Comment:
I would be best to ouput these error mesages in the form next to the fields, or in a specific box in the page. At the moment they are being outputted outside the body tag, which strictly speaking is invalid HTML